src/pages/Leaderboard.tsx

```diff
@@ -1,10 +1,37 @@
 import '../styles.css'
+import { useEffect, useState} from "react"
 
 function Leaderboard() {
 
+    const [users, setUsers] = useState([])
+    const [loading, setLoading] = useState(false)
```

Comment:
We could remove the loading state because we don't use it anywhere. Or we could add a loading indicator based on it